### DISCUSSION TITLE
ERROR	Error: On Vercel, NOENT: no such file or directory, open '/var/task/schema.prisma'

### DISCUSSION THREAD

**Discussion by wahhabb** [prisma/prisma#9806] [Q&A]
Hi! I'm new to Prisma. I modified SvelteKit's demo app to use Prisma, using MySQL. When I do `npm run build` and then `npm run preview`, it works fine on my local machine. But when installed on Vercel, going to the TODO page gives me `JSON.parse: unexpected character at line 1 column 1 of the JSON data`, and my log shows the results below. Any guidance would be greatly appreciated. Thanks!
`[GET] /todos.json
22:21:55:88
2021-10-16T05:21:56.545Z	undefined
	ERROR	Error: ENOENT: no such file or directory, open '/var/task/schema.prisma'
    at Object.openSync (fs.js:498:3)
    at Object.readFileSync (fs.js:394:35)
    at new LibraryEngine (/var/task/index.js:31395:46)
    at PrismaClient2.getEngine (/var/task/index.js:43833:20)
    at new PrismaClient (/var/task/index.js:43802:33)
    at Object.<anonymous> (/var/task/index.js:46480:14)
    at Module._compile (internal/modules/cjs/loader.js:1072:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1101:10)
    at Module.load (internal/modules/cjs/loader.js:937:32)
    at Function.Module._load (internal/modules/cjs/loader.js:778:12) {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: '/var/task/schema.prisma',
  clientVersion: '3.2.1'
}`

**Comment by nenadfilipovic** [comment]
You are trying to parse `todos.json` which does not exist maybe?

**Reply by wahhabb** [reply]
Thank you for responding. This is of course true--the place where the 500 is being thrown is the code "body = await prisma.todo.findMany();" which is what is converted to todos.json. The question is why this fails on Vercel when it works locally.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
